packages/jest-reporters/src/GitHubActionsReporter.ts

```diff
@@ -26,8 +33,73 @@ type AnnotationOptions = {
 
 const titleSeparator = ' \u203A ';
 
+type performaceInfo = {
```

Comment:
Please use CamelCase for types (and this one also has a typo):

```suggestion
type PerformanceInfo = {
```

Reply:
Fixed